Yona/Yona/Challenges/TimeFrameTimeZoneChallengeViewController.swift

```diff
@@ -242,6 +246,8 @@ extension TimeFrameTimeZoneChallengeViewController {
                 self.setChallengeButton.alpha = 0.5
             }
             self.tableView.endUpdates()
+            
+            NSTimer.scheduledTimerWithTimeInterval(0.1, target: self, selector: Selector.tableReload, userInfo: nil, repeats: false)
```

Comment:
I tried it without the times a couple of days ago.

Also its in a added(backlog) for future improvement.